applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -494,6 +505,9 @@ static void cmd_send(struct k_work *work)
 	} else if (err == -ESHUTDOWN) {
 		/*Entered IDLE or UART Power Off */
 		return;
+	} else if (err == -EINPROGRESS) {
+		/* Data mode under processing */
```

Comment:
What about using in_datamode to check the datamode state instead of refering to -EINPROGRESS?

Reply:
You're right, thanks for the advice.